flink-connector-directload/src/main/java/com/oceanbase/connector/flink/OBKVDirectLoadDynamicTableSinkFactory.java

```diff
@@ -0,0 +1,99 @@
+/*
+ * Copyright 2024 OceanBase.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.oceanbase.connector.flink;
+
+import com.oceanbase.connector.flink.sink.OBKVDirectLoadDynamicTableSink;
+import com.oceanbase.connector.flink.utils.OptionUtils;
+
+import org.apache.flink.api.common.RuntimeExecutionMode;
+import org.apache.flink.configuration.ConfigOption;
+import org.apache.flink.configuration.ExecutionOptions;
+import org.apache.flink.configuration.TaskManagerOptions;
+import org.apache.flink.table.catalog.Column;
+import org.apache.flink.table.catalog.ResolvedSchema;
+import org.apache.flink.table.connector.sink.DynamicTableSink;
+import org.apache.flink.table.factories.DynamicTableSinkFactory;
+import org.apache.flink.table.factories.FactoryUtil;
+
+import java.util.HashSet;
+import java.util.Map;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+/** The factory of direct-load dynamic table sink. see {@link DynamicTableSinkFactory}. */
+public class OBKVDirectLoadDynamicTableSinkFactory implements DynamicTableSinkFactory {
+
+    public static final String IDENTIFIER = "obkv-directload";
```

Comment:
oceanbase-directload?  I'm not sure if it is appropriate, what do you think?

Reply:
Renamed.